server/appservices/oauth2.go

```diff
@@ -39,6 +42,7 @@ func (a *AppServices) StoreOAuth2User(appID apps.AppID, actingUserID string, ref
 	if err = a.ensureFromUser(actingUserID); err != nil {
 		return err
 	}
+	a.mm.Frontend.PublishWebSocketEvent(config.WebSocketEventRefreshBindings, map[string]interface{}{}, &model.WebsocketBroadcast{UserId: actingUserID})
```

Comment:
we should do it only if the value has changed

Reply:
Thanks for explaining the reasoning in the meeting today.  I will make the adjustment in the PR and add the WIP label for now.  I believe there is a `changed` bit that I can use to determine if a value has changed.

Reply:
I committed a change that compares the unmarshalled values before publishing the websocket event.  Is there a preference for comparing the bytes instead?